docs/api-table.js

```diff
@@ -792,18 +809,28 @@ export const vue = {
       ['as', 'string', 'form', 'The DOM element to emit for the wrapper'],
     ],
   },
+  Slider: {
+    required: [],
+    props: [
+      ['max', 'number', '100', 'The greatest value in the range of permitted values'],
+      ['min', 'number', '0', 'The lowest value in the range of permitted values'],
+      ['step', 'number', '', 'Specifies the value granularity'],
+      ['label', 'string', '', 'String value for aria-label the slider'],
```

Comment:
Minor one, the description is a bit strange. Maybe we can skip `the slider`? Or add `of` before it? Or is it just me? Feel free to skip if it doesn't make sense

```suggestion
      ['label', 'string', '', 'String value for aria-label'],
```
```suggestion
      ['label', 'string', '', 'String value for aria-label of the slider'],
```

Reply:
yeah, I agree! When I copied these from Fabric, I was thinking that we'll need to go through and improve the descriptions at some point. But I'll change those you mentioned now anyway:)

Reply:
Yea, we will for sure need to go through it all again 👍